gammapy/scripts/image_pipe.py

```diff
@@ -101,6 +102,39 @@ def bkg_map(self, bkg_norm=True):
 
         self.maps["bkg"] = bkg_map
 
+    def make_1d_expected_counts(self, spectral_index=2.3, for_integral_flux=False):
+        """Compute the 1D exposure table for one observation for an offset table
+
+        Parameters
+        ----------
+        spectral_index : float
+            Assumed power-law spectral index
+        for_integral_flux : bool
+            True if you want that the total excess / exposure gives the integrated flux
+
+        Returns
+        -------
+        offset_tab: `numpy.array`
```

Comment:
I think it would be nicer if you return a `Table` with columns `offset` and `npred` instead of two arrays separately.